### PR TITLE
Add summertime CAPE case

### PR DESCRIPTION
**Pull request by XiaSun-Atmos**
This PR adds the 2020 July CAPE case to illustrate the CAPE summertime bias in GFS.

- A minor fix to Lorenzo cast text
- updated index.rst and added 2020CAPE.rst
- added images/2020CAPE directory

### CODE REVIEW

File: docs/source/2020CAPE.rst [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+This case illustrates the low Convective Available Potential Energy (CAPE) issue during summertime in GFS.v16 identified by NOAA Environmental Modeling Center (EMC) Model Evaluation Group (MEG).
+
+..............................
+Model Configuration and Datasets
+..............................
+.. tabs::
+  .. group-tab:: MRW.v1.0
+
+    The case runs are initialized at 00z Jul 23, 2020 with 24 hours forecasting. The corresponding namelist options that need to be changed are listed below. The app uses ``./xmlchange`` to change the runtime settings. The settings that need to be modified to set up the start date, start time, and run time are listed below.
```
Comment (climbfuji) [inline docs/source/2020CAPE.rst]: 24 hours forecast length ?

File: docs/source/2020CAPE.rst [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+    .. code-block:: bash
+ 
+      ./xmlchange RUN_STARTDATE=2020-07-23,START_TOD=0,STOP_OPTION=nhours,STOP_N=24
+
+
+    Initial condition (IC) files are created from GFS operational dataset in NEMSIO format. The `GFS reanalysis dataset <https://www.ncdc.noaa.gov/data-access/model-data/model-datasets/global-forcast-system-gfs>`_ are used as 'truth' to compare with simulation results.
```
Comment (climbfuji) [inline docs/source/2020CAPE.rst]: dataset is singular, thus "is used as truth"

File: docs/source/2020CAPE.rst [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+
+
+      .. container:: sphx-glr-download sphx-glr-download-python
+
+        :download:`Download initial condition files: 2020072300.gfs.nemsio.tar.gz <https://ufs-case-studies.s3.amazonaws.com/2020072300.gfs.nemsio.tar.gz>`
+  .. group-tab:: GFS.v16.0.10
+
+    The GFS model EMC global workflow points to the most up-to-date GFS model development code. The GFS.v16.0.10 is tested in C768 (~13km) resolution and in 128 vertical levels. It uses two scripts, ``setup_expt_fcstonly.py`` and ``setup_workflow_fcstonly.py`` to set up the mode simulation date and case directories.
+
+    The case runs are initialized at 00z Jul 23, 2020 with 24 hours forecasting. The settings that need to be modified to set up the start date and directories are listed below.
```
Comment (climbfuji) [inline docs/source/2020CAPE.rst]: same, forecast length.

**Review by climbfuji — CHANGES_REQUESTED** [review]

File: docs/source/2020CAPE.rst (hunk 1 above)
Comment (XiaSun-Atmos) [reply docs/source/2020CAPE.rst]: Yes, it was validated at f24.

File: docs/source/2020CAPE.rst (hunk 2 above)
Comment (XiaSun-Atmos) [reply docs/source/2020CAPE.rst]: Thank you for catching that. fixed

File: docs/source/2020CAPE.rst (hunk 1 above)
Comment (climbfuji) [reply docs/source/2020CAPE.rst]: Sorry, I was referring to the language. Please use `with a 24h forecast length` instead of `with 24 hours forecasting`.

Comment (XiaSun-Atmos) [reply docs/source/2020CAPE.rst]: Ah, right! Fixed now. Thank you Dom!

**Review by climbfuji — APPROVED** [review]

**Pull request merged**